core/admin.py

```diff
@@ -12,36 +12,46 @@
 from . import tasks
 
 
-def _text_preview(log_file: FileField):
+def _text_preview(target_file: FileField, mimetype, show_end):
```

Comment:
We might want default values for `mimetype` and `show_end`, so callers don't have to specify them if they are the default, e.g. `def _text_preview(target_file: FileField, mimetype: string=None, show_end: bool=True):`.